### PR TITLE
chore: fix imports and unit tests

### PR DESCRIPTION
**Pull request by crenshaw-dev**
Signed-off-by: Michael Crenshaw <michael@crenshaw.dev>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by alexmt** [comment]
Thank you!

**Review by alexmt — APPROVED** [review]
LGTM

**Review by jessesuen — APPROVED** [review]

File: controller/state_test.go [hunk 1]
```diff
@@ -289,7 +289,7 @@ func TestSetHealth(t *testing.T) {
 
 	compRes := ctrl.appStateManager.CompareAppState(app, &defaultProj, "", app.Spec.Source, false, false, nil)
 
-	assert.Equal(t, compRes.healthStatus.Status, health.HealthStatusHealthy)
+	assert.Equal(t, health.HealthStatusHealthy, compRes.healthStatus.Status)
```
Comment (crenshaw-dev) [inline controller/state_test.go:292]: Order was wrong. It's expected, actual.

File: controller/state_test.go [hunk 2]
```diff
@@ -266,7 +266,7 @@ func TestSetHealth(t *testing.T) {
 	app := newFakeApp()
 	deployment := kube.MustToUnstructured(&v1.Deployment{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "apps/v1beta1",
+			APIVersion: "apps/v1",
```
[the remark this answers is not in the document]
Comment (crenshaw-dev) [reply controller/state_test.go:269]: I found it by inspecting `compRes` and noticed the sync status said something about "invalid GVK". I'm surprised other tests that use `apps/v1beta1` are passing.

**Pull request merged**